components/cascader/index.tsx

```diff
@@ -97,6 +98,8 @@ export interface CascaderState {
   flattenOptions: CascaderOptionType[][] | undefined;
 }
 
+const defaultLimit = 50;
```

Comment:
Please document this variable in comments as it is too far from lines where it is used.